src/Bridge/Doctrine/EventListener/PurgeHttpCacheListener.php

```diff
@@ -97,7 +97,7 @@ public function onFlush(OnFlushEventArgs $eventArgs)
     /**
      * Purges tags collected during this request, and clears the tag list.
      */
-    public function postFlush()
+    public function onKernelTerminate()
```

Comment:
indeed good point. there would be a better place if the idea is thought to be good.